services/spar/test-integration/Test/Spar/Scim/UserSpec.hs

```diff
@@ -1081,6 +1109,9 @@ testUpdateRequiresUserId = do
   updateUser_ (Just tok) Nothing user (env ^. teSpar)
     !!! assertTrue_ (inRange (400, 499) . statusCode)
 
+testScimEmailUpdate :: EmailSource -> TestSpar ()
+testScimEmailUpdate _ = pure ()
```

Comment:
```suggestion
testScimEmailUpdate _ = liftIO pending
```

(sorry, my bad)

Reply:
Removed this test, because it will be implemented as part of https://wearezeta.atlassian.net/browse/SQSERVICES-376